terraform_compliance/steps/then/its_value_condition_be_null.py

```diff
@@ -0,0 +1,39 @@
+from terraform_compliance.common.error_handling import Error
+from terraform_compliance.common.helper import Null
+import re
+
+
+def its_value_condition_be_null(_step_obj, condition):
+    if condition not in ['must', 'must not']:
+        Error(_step_obj, 'Condition must be one of "must" or "must not"')
+        return
+
+    condition_must = condition == 'must'
+
+    match = _step_obj.context.match
+    regex = r'{}'.format(u'(\x00|^$)$')
+    is_null = lambda x: x in ([], {}, None, Null) or (isinstance(x, str) and match.regex_match(regex, x))
+
+    for resource in _step_obj.context.stash:
+        values = resource.get('values', {})
+
+        values_is_null = is_null(values)
+        if (values_is_null and not condition_must) or (not values_is_null and condition_must):
+            Error(_step_obj, '{} property in {} is considered to be {}. It is set to {}.'.format(_step_obj.context.property_name, 
+                                                                                                resource.get('address', _step_obj.context.name),
+                                                                                                'Null' if values_is_null else 'not Null',
+                                                                                                values))
```

Comment:
For the error message:
can also put them both at the same time but feels a bit cluttery
e.g. ` _step_obj.context.name, resource.get('address', 'address could not be resolved')`